python/lsst/ts/standardscripts/maintel/home_both_axes.py

```diff
@@ -107,8 +115,3 @@ async def run(self):
         elapsed_time = end_time - start_time
 
         self.log.info(f"Homing both axes took {elapsed_time:.2f} seconds")
-
-        if not self.ignore_m1m3:
```

Comment:
if you disabled it, you need to enable it back here.

Reply:
Not sure I follow. ignore_m1m3 it's basically removed. So there is no "ignore_m1m3" property.

Reply:
but we added the `disable_m1m3_force_balance` option....

Reply:
I think we can remove it.